consensus/types/src/graffiti.rs

```diff
@@ -57,7 +57,7 @@ impl FromStr for GraffitiString {
     type Err = String;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        if s.as_bytes().len() > GRAFFITI_BYTES_LEN {
+        if s.len() > GRAFFITI_BYTES_LEN {
```

Comment:
This one is interesting. String length is in bytes, so makes sense:

https://doc.rust-lang.org/std/primitive.str.html#method.len

Reply:
Yeah i was confused as well, but the lint explicitly stated that they are equivalent